buildkit.gemspec

```diff
@@ -19,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.0'
+  spec.required_ruby_version = '>= 2.1'
```

Comment:
Should we change this here, or should we instead change it in `.rubocop.yml`? I vote for changing it in `.rubocop.yml` as being safer, unless the project has decided to move to ruby 2.1. Thoughts?

Reply:
I believe we don't support ruby 2.0. In Travis CI we only run tests for ruby version 2.1 and 2.2
<img width="1162" alt="screen shot 2018-05-10 at 5 35 01 pm" src="https://user-images.githubusercontent.com/13983801/39895521-97bbef9c-5478-11e8-8a16-4936b6c3069f.png">
And if we do still support ruby 2.0, I believe we should stop supporting it, because it's [EOL date: 2016-02-24](https://www.ruby-lang.org/en/downloads/branches/)

Reply:
https://github.com/Shopify/buildkit/blob/master/.travis.yml#L3-L5

Reply:
Yeah, 2.1 is EOL as well. So you can go with `>= 2.2`